src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private BufferedReader bufferedReader;
+    private String[] arrayToWriteIntoFile = new String[3];
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        try {
+            bufferedReader = new BufferedReader(new FileReader(fromFileName));
```

Comment:
it's better to use try-with-resources struct, thus you imrove secureness of work with memory 
![image](https://github.com/mate-academy/jv-work-with-file/assets/73041091/936dfce8-8e17-436d-9acc-1e5e48450d08)